validator/package.json

```diff
@@ -4,7 +4,8 @@
 	"scripts": {
 		"build": "rimraf dist && tsc",
 		"dev": "tsx src/validator.ts",
-		"check": "biome check"
+		"check": "biome check",
+		"test": "echo \"Error: no test specified\" && exit 0"
```

Comment:
I use `exit 0` so that the test is not failing the workflow runs right now (which would block it from making runs required).